documentation/table_research.md

```diff
@@ -0,0 +1,73 @@
+# Table Research

```

Comment:
This file duplicates a lot of info that's in the architecture doc. We either need to:
- move some of this info over to the architecture doc 
- or just link out to this one from the architecture doc and remove the related content in the architecture doc.